changelog.d/6079.wip.change

```diff
@@ -0,0 +1 @@
+App Layout: Edit layout experiment
```

Comment:
This one has become `.wip.change` 😅
And maybe the other 3 should also be `.wip`?

Reply:
I was definitely tired when I updated the file extension 😅

Reply:
Only this one is WIP btw